pkg/stack/unwind/unwind_table.go

```diff
@@ -29,6 +29,7 @@ import (
 var (
 	ErrNoFDEsFound            = errors.New("no FDEs found")
 	ErrEhFrameSectionNotFound = errors.New("failed to find .eh_frame section")
+	ErrNoRegisterFound        = errors.New("failed to find register for architecture")
```

Comment:
This error is not handled anywhere, it's used more like a "default error" state, what about having that as a string directly rather than an error?

Reply:
Also now that I think about it it should be more like "architecture not supported", no?

Reply:
Good catch! I do feel we should capture an error here and just log it for easier debugging later but open to other suggestions :D